roles/openshift_excluder/tasks/status.yml

```diff
@@ -17,37 +17,37 @@
   register: docker_excluder_installed
   failed_when: false
 
-- name: Update to latest excluder packages
-  package:
-    name: "{{ openshift.common.service_type }}-excluder"
-    state: latest
-  when:
-  - "{{ openshift_excluder_installed.installed_versions | default([]) | length > 0 }}"
-  - not openshift.common.is_containerized | bool
-
-- name: Update to the latest docker-excluder packages
-  package:
-    name: "{{ openshift.common.service_type }}-docker-excluder"
-    state: latest
-  when:
-  - "{{ docker_excluder_installed.installed_versions | default([]) | length > 0 }}"
-  - not openshift.common.is_containerized | bool
+- block:
+  - name: Update excluder packages if requested
```

Comment:
After discussing with @brenton we came to the conclusion that the chances of someone having the wrong repo enabled is high enough that we'd like to make sure that we only upgrade to the latest for the version corresponding to the given upgrade playbook. ie: playbooks/byo/openshift-cluster/ugprades/v3_4/upgrade.yml would only upgrade to the latest atomic-openshift-excluder-3.4* I'm not certain that the package module can handle a glob though and we'd need to account for origin versus OCP versions.